.pre-commit-config.yaml

```diff
@@ -53,9 +60,11 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.1.0
     hooks:
-      - id: trailing-whitespace
-      - id: end-of-file-fixer
       - id: check-added-large-files
+      - id: end-of-file-fixer
+      - id: requirements-txt-fixer
+        args: [requirements-dev.txt, requirements-docs.txt]
```

Comment:
Did you find you needed to pass these files explicitly? I think what will happen now, is that they are always checked no matter if they are changed or not. which I guess is fine - it is lightning fast anyhow I guess and won't probably cause issues. It does require us to explicitly list what we update. It can of course be kind of nice in a way, but also is something that can break as files are renamed etc.

Reply:
I think we have to pass files explicitly.
https://github.com/pre-commit/pre-commit-hooks/blob/master/pre_commit_hooks/requirements_txt_fixer.py#L135
Actually, if we rename a file, this will break, and we will have to update this, but I guess it's only 2 files and it should be ok.

Reply:
@mathbunnyru hmmm but pre-commit is passing arguments of the changes files by default I think, so it will be passed information. I think requirements-txt-fixer just ignores files it doesn't want to work on.

Reply:
Example: https://github.com/jupyterhub/jupyterhub/blob/454e356e4dbfa8e291212f5eaff18dc0c14f5e11/.pre-commit-config.yaml#L39-L45

Reply:
This is a valid point, I will do some quick-and-dirty tests :)